packages/@expo/metro-config/src/transform-plugins/import-export-plugin.ts

```diff
@@ -31,154 +31,129 @@ export type Options = Readonly<{
   };
 }>;
 
+class MapToArray<K, V> extends Map<K, V[]> {
+  public getOrDefault(key: K): V[] {
+    return this.get(key) ?? (this.set(key, []).get(key) as V[]);
+  }
+}
+
+class MapToFirst<K, V> extends Map<K, V> {
+  public setFirst(key: K, value: V): this {
+    if (!this.has(key)) {
+      this.set(key, value);
+    }
+    return this;
+  }
+}
```

Comment:
stylistic nit: it's not a lot, but having these as utility functions may be easier to follow and refactor

Reply:
Make sense, added in https://github.com/expo/expo/pull/39083/commits/d51136072f3c40d7468a3cad1d398807e90a0c28